## This questions are due next discussion section (Week6).

1. Describe the Biology of your project? (one paragraph max). 

Microbial gut communities play an important role in the health of all complex organisms. In terrestrial vertebrates, gut community composition is shaped by the phylogeny and ecology of their hosts (Sullam et al., 2012). However, we know little about gut microbiomes in the largest group of vertebrates—fish, particularly, coral reef fish. As vital components of a healthy coral reef ecosystem, reef fish represent an ideal system to study gut microbiomes within the context of an ecosystem sensitive to unnatural disturbances. Previous studies done by Barber Lab have demonstrated that there is a clear connection between phylogeny and gut micrbiome biodiversity. This program will help us further study how environment, host gender, and host diet impact gut microbiomes of coral reef fish. Creating a program that will take in a lot of data and then output a beta diversity graph. The graphic visual can help us understand the extent to which interactions with host diet and host gender are involved.

2. Describe the kind of data that you think you will be handling (one paragraph max).

The data that we will be working with are DNA sequences from a sample set of 20 fish of the species Chlororus spilurus, which are part of the wrasse family. The sample set includes 5 terminal males and 15 females. The DNA sequenc that was amplified in order to study the microbial gut community of the coral reef fish was the V4 variable region of the 16s rRNA gene.

3. What would you like to do with that data? (one paragraph max).

The raw DNA sequences that we will be working with also include index sequences and primer sequences. We want to clean the data so that we are only left with the DNA sequence corresponding to the V4 variable region of the 16s rRNA gene. Thane we want to use the data to create a graph that creates a beta diversity graph based on ASVs.

4. What will be the output? (one paragraph max).

The final output should be a beta diversity graph that compares the similarities and dissimilarities of the gut microbiomes of each sample.

5. Is there any computational tool, package, etc. you would like to use? What would you do with that? For instance, you could mention shell scripts with `sed` or `awk` and will change the format of a table by replacing `,` with tabs. 
Also, you could mention tools that we haven’t seen in class yet but you herd that may be useful for your project. 
For instance, you could say 
***I will use the package “GenomeGraphs” implement in the R software to visualize genome data***. (a few sentences max).

As of right now, we will be using QUIIME 2 and QUIIME View.

6. What is your biggest concern regarding your project? (a few sentences max).

My biggest concern for the project is confronting an issue with the code and not being able to find a resolution by the time that it is due. I also want the code to be efficent, but also easy enough to do for the common user and someone who has never coded before, which can be a challenge in itself.

## Grade Discussion 5 (week5): 20/20 Nice work!!! :thumbsup:

| **Rubric** | **5pt** | **4pt** | **3pt** | **2pt** | **1pt** |
| --- | ---| --- | --- | --- | --- |
| In-Lab work | **X** | | | |
| Questions | **X** | | | |
| Project README.md | **X** | | | |
| Challenges - Timely Submission | **X** | | | |

